### DISCUSSION TITLE
1.5.1-beta7

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#545] [Обновления]
## Список изменений
- Исправлена работа YouTube Embed через JS API (#544)
- Исправлена работа Bitchute от @SashaXser в #531 (#542)
- Добавлена поддержка Egghead от @SashaXser в #531
- Добавлена поддержка Youku от @SashaXser в #531
- Улучшена логика инициализации БД от @SashaXser в #531
- Добавлено определение языка в VK по субтитрам и обновление зависимостей от @SashaXser в #531
- Убрана устаревшая логика fullscreen от @SashaXser в #531
- Webpack-Userscript изменен на Webpack-Monkey

## What's Changed
* Добавлена поддержка 2 сайтов by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/531

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.5.1-beta6...1.5.1-beta7

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.1-beta7'>1.5.1-beta7</a>.</em>

**Comment by NullVerdict** [comment]
@ilyhalight translate.toil.cc опять умер
![изображение](https://github.com/ilyhalight/voice-over-translation/assets/24498484/9c961f31-37a8-4fb1-8bd0-cdc0a9a2136e)

**Reply by ilyhalight** [reply]
> @ilyhalight translate.toil.cc опять умер ![изображение](https://private-user-images.githubusercontent.com/24498484/309866783-9c961f31-37a8-4fb1-8bd0-cdc0a9a2136e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk1ODE4NjcsIm5iZiI6MTcwOTU4MTU2NywicGF0aCI6Ii8yNDQ5ODQ4NC8zMDk4NjY3ODMtOWM5NjFmMzEtMzdhOC00ZmIxLThiZDAtY2RjMGE5YTIxMzZlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzA0VDE5NDYwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTczNWVkMmNmZjYwOTc2ZTk2MTE1MDJmNTlhZGU1MTdmN2JlYzIwM2JjNDdhOGQzYWRhMTZmNWQ1NTc1Y2EzZTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.cVKmBM7jFBdCqGvekpy8fYi_gWgR5-_7Mdd9Evw82Y8)

Поднял, почему-то из-за либы hyperdx сервер уходил в таймаут. Пока отключил ее

**Reply by NullVerdict** [reply]
Что-то аптайм не очень стабильный... 

![изображение](https://github.com/ilyhalight/voice-over-translation/assets/24498484/94215777-b249-4a36-894e-781ca59ca3c3)
